pom.xml

```diff
@@ -178,6 +178,11 @@
         <artifactId>commons-io</artifactId>
         <version>${commons-io.version}</version>
       </dependency>
+      <dependency>
+        <groupId>org.apache.commons</groupId>
+        <artifactId>commons-lang3</artifactId>
+        <version>${commons-lang3.version}</version>
+      </dependency>
```

Comment:
I've moved `commons-lang3` to `<dependencyManagement>` so a newer version is selected if it uses in transitive dependencies